lib/views/after_auth_screens/events/explore_events.dart

```diff
@@ -121,7 +122,7 @@ class ExploreEvents extends StatelessWidget {
                                           .onPrimary,
                                       child: Container(
                                         padding: const EdgeInsets.symmetric(
-                                          horizontal: 20,
+                                          horizontal: 16,
```

Comment:
Don't use hard coded value it will break on different screen size. Use sizeConfig. Change all the hard coded value to sizeConfig in this specific file.